src/utils/userActive.ts

```diff
@@ -0,0 +1,33 @@
+import { onMounted, onUnmounted, ref, watch } from "vue";
+
+export const useUserActive = () => {
```

Comment:
Go for https://vueuse.org/core/useidle when it works for you

Reply:
Yeah, was writing this on the plane, so had no access to vueuse.org :).